### ISSUE TITLE
SMT-LIB: dolmen fails to recognize `and` with only one argument

### ISSUE THREAD

**Issue by hansjoergschurr** [Gbury/dolmen#57]
The attached file does not produce an error, but in line 57 an `and` operator is only applied to one argument:
```
 (let (($x159 (and (and (fp.eq x!FP (fp.add roundNearestTiesToEven xFP ?x151)) $x157))))
```

[problem.smt2](https://github.com/Gbury/dolmen/files/6199701/problem.txt)

**Comment by Gbury** [comment]
If I understand correctly, this example should fail because the `and` function should take at least two arguments, is that correct ?

**Comment by hansjoergschurr** [comment]
yes! `and` is a binary operator with the extra argument `:left-assoc` which makes it possible to apply it to more than one argument.

**Comment by Gbury** [comment]
Indeed, this has been on my radar for some time; I'll try and fix it as soon as possible.

**Comment by Gbury** [comment]
Fix proposed in #61
